components/ui/label.tsx

```
@@ -1,26 +1,22 @@
-))
-Label.displayName = LabelPrimitive.Root.displayName
+const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
+  ({ className, ...props }, ref) => (
+    <label
+      ref={ref}
+      className={cn(
+        "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70",
+        className
+      )}
+      {...props}
+    />
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding an accessible text content to the label element.
Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)